src/features/events/models/EventDataModel.ts

```diff
@@ -34,18 +35,62 @@ export default class EventDataModel extends ModelBase {
     this._repo = new EventsRepo(env);
   }
 
+  getAvailParticipants(): number {
+    const participants = this.getParticipants().data;
+    return participants ? participants.length : 0;
+  }
+
   getData(): IFuture<ZetkinEvent> {
     return this._repo.getEvent(this._orgId, this._eventId);
   }
 
+  getParticipantStatus = (
+    availParticipants: number,
+    reqParticipants: number
```

Comment:
Why have these numbers as arguments? Couldn't the model just get the values using `this.getAvailParticipants()` and `this.getData().data.num_participants_required`?